client/app/scripts/superdesk-authoring/views/article-edit.html

```diff
@@ -37,12 +37,14 @@
 
 <div class="field" ng-if="schema.byline">
   <label translate>By</label>
+  <!-- Placeholder text is temporary, ideally it should come from content type -->
   <input type="text"
          class="byline"
          id="byline"
          ng-model="item.byline"
          ng-change="autosave(item)"
-         ng-if="_editable">
+         ng-if="_editable"
+         placeholder="Add text here">

```

Comment:
Because ideally placeholders will come from field definitions, I just added generic placeholders for now.